packages/terra-time-input/src/TimeInput.module.scss

```diff
@@ -311,4 +311,10 @@
       }
     }
   }
+  // stylelint-disable property-no-vendor-prefix
+  input::-webkit-outer-spin-button,
```

Comment:
why was this added? Was it to suppress warning?

Reply:
This style was included to hide arrow icons in input field when we use type as number, it was to suppress property-no-vendor-prefix warning